screens/menu-screen/index.js

```diff
@@ -80,6 +80,19 @@ class MenuScreen extends Component {
                     >
                         <Text style={styles.buttonText}>Feedback Form</Text>
                     </TouchableOpacity>
+                    <TouchableOpacity
+                        style={styles.button}
+                        onPress={() => {
+                          if(Platform.OS == 'ios'){
```

Comment:
you'll need to add 
```suggestion
                          import { Platform } from 'react-native'
```
at the top of this file, otherwise, when clicking the Review this App button the app crashes with ```Can't find variable: Platform```

Reply:
@smaraf, Sorry about this. I thought i checked this before creating the PR.

Reply:
No worries. It works great now.